ghcide/src/Development/IDE/Plugin/Completions/Logic.hs

```diff
@@ -266,7 +266,8 @@ mkNameCompItem doc thingParent origName origMod thingType isInfix docs !imp = CI
             let (args, ret) = splitFunTys t
               in if isForAllTy ret
                   then getArgs ret
-                  else Prelude.filter (not . isDictTy) args
+                  -- TODO: Do we want to use multiplicity here?
+                  else Prelude.filter (not . isDictTy) $ map scaledThing args
```

Comment:
This code is only concerned with extracting argument names, so I don't see how multiplicity would be relevant here

Reply:
Ok, that's good. I wasn't really familiar with what the code is doing.